test/sql/test_default_value/T/test_json_strict_validation.sql

```diff
@@ -0,0 +1,189 @@
+-- name: test_json_strict_validation
+-- Test strict JSON validation for default values
+-- using strict JSON parsing (rejects non-standard JSON syntax)
+
+DROP DATABASE IF EXISTS test_json_strict_db;
+CREATE DATABASE test_json_strict_db;
+USE test_json_strict_db;
+
+-- ============================================
+-- Test 1: Valid Standard JSON - Should succeed
+-- ============================================
+CREATE TABLE test_json_strict_valid (
+    id INT,
+    data1 JSON DEFAULT '{"key": "value"}',
+    data2 JSON DEFAULT '{"name": "Alice", "age": 30}',
+    data3 JSON DEFAULT '{}',
+    data4 JSON DEFAULT 'null',
+    data5 JSON DEFAULT '""',
+    data6 JSON DEFAULT '123',
+    data7 JSON DEFAULT 'true'
+) DUPLICATE KEY(id)
+DISTRIBUTED BY HASH(id) BUCKETS 1
+PROPERTIES("replication_num" = "1");
+
+-- Verify: Insert data and query
+INSERT INTO test_json_strict_valid (id) VALUES (1);
+SELECT * FROM test_json_strict_valid;
+
+-- ============================================
+-- Test 2: Unquoted Keys - Should fail
```

Comment:
this kind of syntax check can be put into the UT, which is more efficient, otherwise the integration test will become slower and slower